CHANGES.md

```diff
@@ -1,5 +1,43 @@
 # Changes
 
+## edge-23.4.1
+
+This edge release introduces request-level HTTP circuit-breaking using a
```

Comment:
We probably want to explicitly call this out as a release candidate. There should be other examples of that in this file.

Reply:
good call, done in 11dd950b205d5c31a94d6dd537ce65b9fe1ee6c7!